src/task3/config/global.js

```diff
@@ -0,0 +1,3 @@
+export const SERVER_PORT = 3000;
```

Comment:
Consider sticking to on style of exporting:
1. The "inline exports" `export const smth = 42;`
2. The "barrel exports" `export { smth }`
And try to keep it consistent in your app as it is essentially your modules API.